apps/website/src/pages/ambassadors/index.tsx

```diff
@@ -216,7 +229,7 @@ const AmbassadorsPage: NextPage = () => {
           </div>
         </Section>
       </div>
-
+      <SubNav links={sectionLinks} className="z-20" />
```

Comment:
```suggestion

      <SubNav links={sectionLinks} className="z-20" />

```
Nit: Spacing around sections